components/cards/AdvancedStatCard.tsx

```diff
@@ -1,65 +1,36 @@
 import { CircularProgress } from "@mui/material";
-import { FC, ReactNode, useState } from "react";
-import { Info } from "lucide-react";
+import { FC, useState } from "react";
 import style from "../../styles/AdvancedStatCard.module.css";
 
 interface StatCardProps {
   title: string;
   statValue: number;
   isLoading?: boolean;
-  progress?: string;
-  progressDescription?: string;
-  icon: string;
-  tooltipText: string;
+  icon?: string;
 }
 
 export const AdvancedStatCard: FC<StatCardProps> = ({
   title,
   statValue,
   isLoading,
-  progress,
-  progressDescription,
-  icon,
-  tooltipText
+  icon = "/icons/infoIcon.png",
 }) => {
   const [showTooltip, setShowTooltip] = useState(false);
   return (
     <div className={style.card}>
-      <div className={style.text}>
-        <h3 className={style.subtitle}>{title}</h3>
+      <div className="flex flex-col justify-between w-full gap-3">
+        <div className="flex justify-between items-center w-full">
+          <h3 className={style.title}>{title}</h3>
+          <img src={icon} alt="icon" className={style.icon} />
+        </div>
         {isLoading ? (
           <CircularProgress className={style.loader} />
         ) : (
           <>
             <p className={style.valueLabel}>{statValue}</p>
-            <span
-              className={`${style.progress} ${
-                progress?.startsWith("-") && style.negative
-              }`}
-            >
-              {progress}
-            </span>
-            <label className={style.progressLabel}>{progressDescription}</label>
           </>
         )}
       </div>
-      <div className={style.illustrations}>
-        <div
-            className={style.tooltipWrapper}
-            onMouseEnter={() => setShowTooltip(true)}
-            onMouseLeave={() => setShowTooltip(false)}
-          >
-            <Info className={style.infoIcon} />
```

Comment:
While merging you deleted the tooltip hover. Just fix that and we're good for merging :D